mongodb.js

```diff
@@ -0,0 +1,102 @@
+const { MongoClient, Timestamp } = require("mongodb");
+
+module.exports = { db_connect, db_setup, new_print_job, new_workflow, new_workflow_step, new_simulation_report };
+
+
+async function db_connect(){
+	const uri = "mongodb://localhost:27017/HP";
```

Comment:
Hard coding `uri` is fine for now, but we should put it in a config file to change it. Obviously, we won't be hosting it on `localhost` in the future.